openpbs.spec

```diff
@@ -599,69 +610,3 @@ for i in "${required_pkg[@]}"; do
-- Remove pbspro-rpmlintrc from source list
-- Updates to conditional build of pbspro-ptl package
-* Mon Dec 10 2018 bayucan <bayucan@altair.com> - 1.25
-- Add changelog
-* Wed Nov 28 2018 bayucan <bayucan@altair.com> - 1.24
-- Handle non-conffile-in-etc,explicit-lib-dependency
-* Wed Nov 14 2018 bayucan <bayucan@altair.com> - 1.23
-- Enable pbspro-rpmlintrc under opensuse
-* Thu Oct 25 2018 riyazhakki <riyazhakki@gmail.com> - 1.22
-- Add a M4 macro to enable online data compression in TPP
-* Wed Sep 12 2018 Bhroam Mann <bmann@altair.com> - 1.21
-- Remove reference to ibm-* platforms.
-* Mon Aug 27 2018 Hiren Vadalia <hiren.vadalia@altair.com> - 1.20
-- Add new OSes in CI and remove all old OSes and hacks from Travis
-* Tue Aug 7 2018 Michael Karo <mike0042@gmail.com> - 1.19
-- Build PBS Pro on OpenSUSE Tumbleweed
-* Thu May 10 2018 sandisamp <sandisamp@gmail.com> - 1.18
-- Fix declare not found error in ubuntu and added requires bash in spec file
-* Mon Feb 5 2018 Michael Karo <mike0042@gmail.com> - 1.17
-- Update OSS version from 17.1.0 to 18.1.0
-* Thu Dec 14 2017 Dinesh <dineshjoshi1306@gmail.com> - 1.16
-- Fix copyright headers
-* Tue Sep 26 2017 lisa-altair <lisa@altair.com> - 1.15
-- Remove calling pbs_postinstall on CLE6.0
-* Tue Jun 13 2017 Hiren Vadalia <hiren.vadalia@altair.com> - 1.14
-- Update PBS version to 17
-* Thu Mar 23 2017 Kevin Liu <minghui.liu@trincoll.edu> - 1.13
-- Update copyright year in source file headers
-* Thu Mar 9 2017 Michael Karo <mike0042@gmail.com> - 1.12
-- Update to support installations and upgrades on Cray XC CLE 5.2
-* Thu Jan 5 2017 nithinj <nithin.johnson@altair.com> - 1.11
-- Update as pbs_mom not coming up after switching on compute node on cpuset machines
-* Fri Nov 18 2016 Jan Krcmar <honza801@gmail.com> 1.10
-- Add debian package building support using rpmbuild and alien.
-* Mon Aug 8 2016 Michael Karo <mike0042@gmail.com> - 1.9
-- Update to allow sendmail or postfix to satisfy requirements
-* Mon Aug 1 2016 nithinj <nithin.johnson@altair.com> - 1.8
-- Update so that systemd pbs unit file refer path from pbs.conf file
-* Wed Jun 29 2016 Michael Karo <mike0042@gmail.com> - 1.7
-- Specify libexecdir on SUSE systems
-* Wed Jun 22 2016 nithinj <nithin.johnson@altair.com> - 1.6
-- Replace init script with unit file for systems that support systemd process management
-* Thu Jun 16 2016 Michael Karo <mike0042@gmail.com> - 1.5
-- Handle starting multinode jobs
-* Wed Jun 15 2016 Michael Karo <mike0042@gmail.com> - 1.4
-- Build PBS Pro under non-OHPC OBS instance
-* Fri Jun 3 2016 Nithin Johnson <nithin.johnson@altair.com> - 1.3
-- Add timezone and python-xml as dependencies
-* Wed May 25 2016 Michael Karo <mike0042@users.noreply.github.com> - 1.2
-- Update version to 14.0.1
-* Mon May 23 2016 arungrover <arun.grover@altair.com> - 1.1
-- Change to make sure that unsupported hook files are not compiled and packaged.
-* Thu May 12 2016 Hiren Vadalia <hiren.vadalia@altair.com> - 1.0
-- Initial commit of pbspro
```

Comment:
Why have you removed the changelog? IIRC, rpmlint will complain if there is no changelog present.

Reply:
Since it was outdated so I thought it's not used... I will add it back with saying "Not used anymore" which will make rpmlint happy

Reply:
But it is used, at least by me when I make a change. New entries are added at the top, and there were a few in 2019.

Reply:
Ok, I am confused here, so this changelog is only says about changes in spec file or PBS?
Also why we need to keep history in file when we have commit history?
For rpmlint we can add one changelog saying either "Not used" or "Look into commit history"

Reply:
It's only for changes to the spec file. This is one of those things that is done "by convention" in the wonderful world of RPMs. We didn't have a changelog until rpmlint complained. You'll need to keep at least one entry in the changelog using the format that is currently being followed.